checkQC/parsers/interop_parser.py

```diff
@@ -53,7 +53,8 @@ def run(self):
         py_interop_summary.summarize_run_metrics(run_metrics, summary)
 
         lanes = summary.lane_count()
-        reads = self.get_non_index_reads(summary)
+        reads = [read_nbr for read_nbr in range(summary.size())]
```

Comment:
Since you're getting all the reads now you can probably remove this line and replace line 62 by
```python
for read_nbr in range(summary.size()):
```

Reply:
I have a question on this, kindly correct me as I am still learning. 
I see that on line 62, both the read value (i.e original_read_nbr) and index (i.e new_read_nbr) are been used.
And I think using this; `for read_nbr in range(summary.size()):` will mean we only have the index of the read. Or is the index read same as the value of each read?

Reply:
Oh, yes I now understand that. Thanks for the explanation. I have now made that change.